src/graphing.py

```diff
@@ -2,13 +2,16 @@
 import PyQt5.QtCore as core
 from qfluentwidgets import *
 from pyqtgraph import *
-import src.statistics_test as st
+import statistics_test as st
```

Comment:
This change will break the tests. All modules must be imported under src.modulename